app/routes/builder.tsx

```diff
@@ -12,7 +12,6 @@ import { ClientOnly } from "~/components/ClientOnly";
 import { getOptionalUser, mustBeAllowedToSubmit } from "~/auth.server";
 import { getBartenderTokenByUser } from "~/bartender-client/token.server";
 import { Haddock3WorkflowBuilder } from "~/builder/Form.client";
-import { haddock3Styles } from "~/builder/styles";
```

Comment:
The `app/builder/styles.ts` file is also used at app/routes/jobs.$id.edit.tsx:31:31 .  Very strange even if I remove the styles.ts and the import in edit route the toast still works. It must come with its own inlined style.

Can you remove the styles.ts and the import of it in *edit.tsx route?